lib/airbrake-ruby.rb

```diff
@@ -240,5 +241,23 @@ def close
     def create_deploy(deploy_params)
       @notifiers[:default].create_deploy(deploy_params)
     end
+
+    ##
+    # Returns current thread context. Can be used to set new thread context
+    # values. The thread context is cleared on every `notify/notify_sync` call,
+    # and also and when the current thread `join`'s. The values assigned to the
+    # thread context are passed to the `params` field of a notice.
+    #
+    # @example
+    #   Airbrake.thread_context[:bingo] = :bango
+    #
+    #   # This call will deliver `{ :bingo => :bango }` to the dashboard.
+    #   Airbrake.notify('Oops')
+    #
+    # @since v2.1.0
+    # @return [Airbrake::ThreadContext] the thread context of
+    def thread_context
```

Comment:
@kyrylo `.thread_` feels like "leaking" implementation details to the client. WDYT about only `.context`?